docs/examples/hosting/docker/caddy/Caddyfile

```diff
@@ -0,0 +1,28 @@
+localhost {
+    encode gzip zstd
+
+    handle_path /server* {
+        reverse_proxy * server:3000 {
+            header_up Host {host}
+            header_up X-Real-IP {remote}
+            # header_up X-Forwarded-For {remote}
```

Comment:
This header is commented because it generates a warning/notice, since it's sent by default.